src/shared/i18n.js

```diff
@@ -1,20 +1,37 @@
 // @flow
 import i18n from 'i18next';
```

Comment:
same question: this is probably handled by transifex as well?

Reply:
no, same here, we need to declare the languages we want in the app in both places (here and your other comment).